app-doc/cantera-docs/cantera-docs-2.4.0.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit desktop
+
+DESCRIPTION="Documentation API reference for Cantera package libraries"
+HOMEPAGE="https://cantera.org"
+SRC_URI="https://github.com/band-a-prend/gentoo-overlay/releases/download/ct-docs-${PV}/${P}_modified_menu.tar.xz -> ${P}.tar.xz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+S="${WORKDIR}/"
+
+src_install() {
+	dodir /usr/share/cantera/doc/
+	cp -R "${S}/." "${D}/usr/share/cantera/doc/" || die "cantera-docs installation failed"
+
+	make_desktop_entry "/usr/bin/xdg-open /usr/share/cantera/doc/doxygen/html/index.html" "Cantera Doxygen Documentation" "text-html" "Development"
```

Comment:
Dep on xdg-utils?

Reply:
Do you mean the `inherit xdg-utils` with appropreate `xdg_desktop_database_update` (without `_mimeinfo_update`) within `pkg_postinst()` and `pkg_postrm()` sections?

Reply:
Yes

Reply:
Sorry, no, I didn't read your question right. I mean you need to RDEPEND on xdg-utils package if you're using xdg-open. Regenerating caches isn't necessary here, is it?